.circleci/real_config.yml

```diff
@@ -2920,6 +2920,21 @@ workflows:
               mark: ["stress_test"]
               det-version: [$CIRCLE_SHA1]
 
+      - test-e2e-aws:
+          name: test-e2e-gpu-single-quarantined
+          context: aws
+#          filters:
+#            branches:
+#              only: main
+          requires:
+            - package-and-push-system-local
```

Comment:
The non-quarantined requirement is for `package-and-push-system-dev`. I cargo-culted the requirement here, and changed it to `package-and-push-system-local` because this workflow doesn't have a `-dev`.